WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -148,7 +152,7 @@ export const apiV0Urls = {
     resetClaimCount: (wcaId) => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_wrt_person_reset_claim_count_path("${wcaId}"))%>`,
   },
   userRoles: {
-    listOfUser: (userId, {isActive, isGroupHidden, status}) => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_index_for_user_path('${userId}')) %>?${jsonToQueryString({ isActive, isGroupHidden, status })}`,
+    listOfUser: (userId, sort, {isActive, isGroupHidden, status}) => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_index_for_user_path('${userId}')) %>?${jsonToQueryString({ sort, isActive, isGroupHidden, status })}`,
```

Comment:
Why not include `sort` in the existing object parameter? Then there wouldn't need to be clarifying comments every time this is called.

Reply:
The object parameters are basically the parameters which are part of the role. Since sort is different, this is included separately. Another reason can be to match how `listOfGroup` is initialized.